dags/hivemind_etl_helpers/src/db/gdrive/gdrive_loader.py

```diff
@@ -0,0 +1,42 @@
+from typing import List, Optional
+
+from llama_index.core.schema import Document
+from llama_index.readers.google import GoogleDriveReader
+
+
+class GoogleDriveLoader:
+    def __init__(self, client_config):
+        self.client_config = client_config
+
+    def load_data(
+        self,
+        drive_ids: Optional[List[str]] = None,
+        folder_ids: Optional[List[str]] = None,
+        file_ids: Optional[List[str]] = None,
+    ) -> List[Document]:
+        """Loads documents from Google Drive.
+
+        Args:
+            drive_ids: List of the google drive ids.
+            folder_id: List of the google folder ids.
+            file_ids: List of google drive file ids.
+
+        Returns:
+           List of loaded document objects.
+        """
+
+        loader = GoogleDriveReader(self.client_config)
+
+        all_docs = []
+        if folder_ids:
+            for folder_id in folder_ids:
```

Comment:
Could you add logs for the conditions and the for loops so if the process takes long we would know it is in progress?